daprdocs/content/en/developing-applications/authenticating/authenticating-aws.md

```diff
@@ -0,0 +1,50 @@
+---
+type: docs
+title: "AWS"
+linkTitle: "AWS"
+weight: 10
+description: "Information about authentication and configuration options for AWS"
+---
+
+All dapr components using various AWS services (DynamoDB, SQS, S3, etc) use a standardized set of attributes for configuration, these are described below.
```

Comment:
dapr --> Dapr

(also in other places in this doc, Dapr should always be in capital letters unless showing a bash command)